src/components/util/CodelabCard/index.js

```diff
@@ -25,48 +25,107 @@ const CardComponent = ({
     setLogoPath(org);
   }, [org]);
 
+ const getDates = (str) => {
+        let today = new Date();
+        let currentYear = today.getFullYear();
+        let currentMonth = today.getMonth()+1;
+        let currentDate = today.getDate();
+        let arr = str.split(" ");
+        let ageString = "";
+
+        let givenDate = arr[0];
+        let givenMonth = arr[1];
+        let givenYear = arr[2];
+
+        var months = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"];
```

Comment:
You should stick with `let` and `const`. What is the reason behind using `var` here?

Reply:
https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75

Reply:
@Niweera Thank you for suggesting the precious read. I was aware of the context, but I will try to maintain consistency from now onwards.